crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -295,7 +295,7 @@ NAME:   minmay_lair_entry_lava
 TAGS:   lair_entry no_monster_gen transparent
 ORIENT: float
 WEIGHT: 1
```

Comment:
Could consider upping the weight to 2 or 4 now that it's a significantly easier fight. Either way, the end of an era...